eg/Classes/TaskletDialog.py

```diff
@@ -162,10 +162,10 @@ def ProcessingTask(self, *args, **kwargs):
             self.Configure(*args, **kwargs)
         except:
             eg.PrintTraceback()
+
```

Comment:
if you mean the blank line.... if there was a "blank line" there before and it had been tabbed out.. my IDE will remove any unneeded white space so it will look like an addition..

If memory serves in this case i was modifying the code right in that area so it is possible i added something then deleted it and didn't remove the empty line.. Let me go back and look at the code..